spotify_player/src/cli/handlers.rs

```diff
@@ -178,8 +179,99 @@ pub fn handle_cli_subcommand(
             std::process::exit(1);
         }
         Response::Ok(data) => {
-            println!("{}", String::from_utf8_lossy(&data));
+            println!("{}", String::from_utf8_lossy(&data).replace("\\n", "\n"));
             std::process::exit(0);
         }
     }
 }
+
+fn handle_playlist_subcommand(args: &ArgMatches, socket: &UdpSocket) -> Result<()> {
+    let (cmd, args) = args.subcommand().expect("playlist subcommand is required");
+    let command = match cmd {
+        "new" => {
+            let name = args
+                .get_one::<String>("name")
+                .expect("name arg is required");
+
+            let description = args
+                .get_one::<String>("description")
+                .map(|s| s.to_owned())
+                .unwrap_or_default();
+
+            let public = args.get_flag("public");
+            let collab = args.get_flag("collab");
+            PlaylistCommand::New {
+                name: name.to_owned(),
+                public,
+                collab,
+                description,
+            }
+        }
+        "delete" => {
+            let id = args
+                .get_one::<String>("id")
+                .expect("id arg is required")
+                .to_owned();
+
+            let pid = PlaylistId::from_id(id)?;
+
+            PlaylistCommand::Delete { id: pid }
+        }
+        "list" => PlaylistCommand::List,
+        "import" => {
+            let from_s = args
+                .get_one::<String>("from")
+                .expect("'from' PlaylistID is required.")
+                .to_owned();
+
+            let to_s = args
+                .get_one::<String>("to")
+                .expect("'to' PlaylistID is required.")
+                .to_owned();
+
+            let delete = args.get_flag("delete");
+
+            let from = PlaylistId::from_id(from_s.to_owned())?;
+            let to = PlaylistId::from_id(to_s.to_owned())?;
+
+            println!("Importing '{}' into '{}'...\n", from_s, to_s);
+            PlaylistCommand::Import { from, to, delete }
+        }
+        "fork" => {
+            let id_s = args
+                .get_one::<String>("id")
+                .expect("Playlist id is required.")
+                .to_owned();
+
+            let id = PlaylistId::from_id(id_s.to_owned())?;
+
+            println!("Forking '{}'...\n", id_s);
+            PlaylistCommand::Fork { id }
+        }
+        "update" => {
+            let id_s = args.get_one::<String>("id");
+
+            let pid = if id_s.is_some() {
+                Some(PlaylistId::from_id(id_s.unwrap().to_owned())?)
+            } else {
+                None
+            };
+
+            let delete = args.get_flag("delete");
+
+            if pid.is_some() {
+                println!("Updating '{}'...\n", id_s.unwrap());
+            } else {
+                println!("Updating all imports...\n");
+            }
```

Comment:
The message should make clear that the command updates/syncs imports for a given playlist/ all playlists.

Reply:
nit: it's related to what I mentioned before about `update` command.